APE24.rst

```diff
@@ -0,0 +1,443 @@
++==========================+=============================+============================+
+| Editorial infrastructure | Editor-in-Chief,            | Editor, reviewer           |
+| for a single review      | editor (domain specific),   |                            |
+|                          | 2 reviewers (1 can be       |                            |
+|                          | non-domain specific)        |                            |
++--------------------------+-----------------------------+----------------------------+
+| Editorial team           | `pyOpenSci Editorial Board`_| 2 editors that rotate, see |
+|                          |                             | `Astropy team roles`_      |
++--------------------------+-----------------------------+----------------------------+
+| Review style             | Open review                 | Reviewer is anonymous by   |
+|                          |                             | default                    |
++--------------------------+-----------------------------+----------------------------+
+| Post-review checks       | We will be creating a set of| Astropy has static badges  |
+|                          | checks to see if the package| that need to be manually   |
+|                          | is still maintained and in  | updated.                   |
+|                          | what state. We might be able|                            |
+|                          | to work with Astropy (and   |                            |
+|                          | other communities) on what  |                            |
+|                          | this looks like.            |                            |
++--------------------------+-----------------------------+----------------------------+
+
+.. _pyOpenSci Editorial Board: https://www.pyopensci.org/about-peer-review/#our-editorial-board
+.. _Astropy team roles: https://www.astropy.org/team#roles
+
+As of July 2023, Astropy had the following `pre-APE 24 Affiliated Package review guidelines`_
+and followed this `pre-APE 24 Affiliated Package review process, with templates`_.
+
+.. _pre-APE 24 Affiliated Package review guidelines: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_guidelines.md
+.. _pre-APE 24 Affiliated Package review process, with templates: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_process.md
+
+Meanwhile, pyOpenSci has its own
+`submission guide for package authors <https://www.pyopensci.org/software-peer-review/how-to/author-guide.html>`_,
+`accompanying GitHub issue templates <https://github.com/pyOpenSci/software-submission/issues/new/choose>`_, and
+`listing of accepted packages <https://www.pyopensci.org/python-packages.html#our-accepted-python-open-source-packages>`_.
+As of July 2023, they are in the process of revamping the submission form to handle
```

Comment:
yes so  i've added the peer review link to our packages page already!

<img width="1276" alt="Screen Shot 2023-08-01 at 4 19 58 PM" src="https://github.com/astropy/astropy-APEs/assets/7649194/601f4c59-d0a7-4708-ad94-46c7e30098c1">

but we might want to reword this to not conflate the website (image above) with our submission form (a github template issue).

Reply:
I will update the text to reflect that you have completed the link to reviews (👏) but I am not sure what you mean by "reword this to not conflate the website (...) with our submission form". I did link to your template too, so hope you can clarify. Thanks!

Reply:
@lwasser , can you please clarify on what else I need to change to satisfy "reword this to not conflate the website (...) with our submission form"?

Reply:
@pllim i think we can resolve this. some of me wondered if this was attached to the wrong place or text is gone. i don't see any reference to linking to the reviews on the website in this section of the content. I think originally i must have seem some language around linking to the review from the website mixed in to the review process itself. and those two things are separate. so let's just resolve this comment.